easybuild/easyconfigs/g/GConf/GConf-3.2.6-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,57 @@
+easyblock = 'ConfigureMake'
+
+name = 'GConf'
+version = '3.2.6'
+
+homepage = 'https://projects.gnome.org/gconf/'
+description = """GConf is a system for storing application preferences.
+ It is intended for user preferences; not configuration
+ of something like Apache, or arbitrary data storage."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = ['http://ftp.gnome.org/pub/GNOME/sources/GConf/%(version_major_minor)s/']
+sources = [SOURCE_TAR_XZ]
+patches = [
+    '%(name)s-%(version)s_call-dbus_g_thread_init.patch',
+    '%(name)s-%(version)s_fix_wrong_return_value.patch',
+    '%(name)s-%(version)s_gconf-path-max-hurd.patch',
+    '%(name)s-%(version)s_readd_gconf_engine_key_is_writable.patch',
```

Comment:
@akesandgren Wouldn't it be cleaner to combine all these into a single patch file named `*_bug-fixes.patch`? It would be a single file that's not too big, so easier to handle... Simply `cat`'ing all files together should work, I think.

Reply:
Well, leaving it as is makes it easier to track any new changes that is made.
But if you want it as a single one I can fix that.

I usually prefer patches to be small and single purpose.

Reply:
But in that case we can at least start with multi_dep:ing h5py .... give me a minute.

Reply:
@boegel ping
Really want single patch or ?